### PR TITLE
Remove requirement for jenkins credentials

### PR DESCRIPTION
**Pull request by odyssey4me**
The jenkins credentials are not required to query the jobs
and are somewhat painful to maintain. Rather than use them
to derive an email address, we add the email_from parameter
and remove the use of jenkins credentials entirely.
    
The email_from argument is only required if an email is
being sent

Fixes: #178

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -21,6 +19,7 @@ Create a file named `config.yaml` based off `config.yaml.example` with the follo
 - **certificate**: CRT file to authenticate with Jira server
 - **smtp_host**: SMTP host of your email
 - **email_subject**: Subject of your email report
+- **email_from**: The email address of the sender.
```
Comment (nathan-weinberg) [inline docs/README.md:22]: nit: can we remove the period?

**Review by nathan-weinberg — APPROVED** [review]
One nit but otherwise LGTM!

Going to add some other stakeholders to review before merging

**Comment by nathan-weinberg** [comment]
Also, I did successfully test this with Enterprise DFG jobs

**Review by agarcia64 — APPROVED** [review]
Tested on my machine, everything works as expected and the code looks good to me.

**Review by amodi5 — APPROVED** [review]
LGTM

File: docs/README.md (hunk 1 above)
Comment (odyssey4me) [reply docs/README.md]: Done.

**Pull request merged**